08_cell_line_prediction/nbconverted/lasso_range_all.py

```diff
@@ -0,0 +1,374 @@
+#!/usr/bin/env python
```

Comment:
Since this file and lasso_range_all.py exist in several directories with seemingly minimal changes (at least between this one and the one in 02_cancer_type_classification), would it be possible to combine them? Maybe not with the way that the notebooks work, but there seems to be enough shared material here that it may be worth putting them out into a functions file so if you make changes, you only have to change it in one analysis.

Reply:
On further reflection, a lot of the inner workings are put in other functions, so ignore me.